kitchen.yml

```
@@ -17,12 +17,23 @@ platforms:
   - name: debian-9
   - name: fedora-latest
   - name: freebsd-11
-  - name: freebsd-12
+  - name: freebsd-12.2 # freebsd-12 bento box is out of sync with the ports tree
```

I hate this yamllint check but this should fix it.
```suggestion
  - name: freebsd-12.2  # freebsd-12 bento box is out of sync with the ports tree
```